fix(station): bump ic-cdk to fix canister_status parsing

jedna opened the pull request.

jedna commented on apps/wallet/src/pages/ExternalCanisterDetailPage.vue:
```diff
@@ -59,7 +59,8 @@
                   wasm_memory_limit: canisterDetails.status.value.settings.wasm_memory_limit,
-                  log_visibility: canisterDetails.status.value.settings.log_visibility,
+                  log_visibility: canisterDetails.status.value.settings
+                    .log_visibility as LogVisibility,
```
I intentionally did not add it as it seems more like a feature for another PR than a fix. But I'm not fully aware of the UI. Is it required?

keplervital replied:
We display that and allow the users to change in the UI, so that cast will break that UI when users would have set variant.

![Screenshot 2025-03-12 at 09 42 08](https://github.com/user-attachments/assets/b59d4444-cde0-46e6-944e-fe103f9e6040)

keplervital replied:
as discussed this PR should show the variant as unsupported and a follow up PR will be created to add the support for it

mraszyk commented:
Should we add a regression test testing the case of calling canister_status on the station for a canister with log visibility restricted to a set of principals?

jedna commented:
I updated the PR to display the disabled "Allowed viewers (unsupported)" option for canisters (disregard screenshot copy) with this value and the former 2-option select box for other canisters.

I also added the regression test to cover the behavior with these two canister_status response variants. 

Another PR will come later to support allowed viewers in the UI fully.

<img width="815" alt="Screenshot 2025-03-12 at 12 20 53" src="https://github.com/user-attachments/assets/3a2a6420-9d59-4f3d-8e24-a8d3eded01b2" />
<img width="819" alt="Screenshot 2025-03-12 at 12 22 47" src="https://github.com/user-attachments/assets/9bf4daaf-6dbf-45ff-a517-0d55c8ff5887" />

mraszyk commented:
> I also added the regression test to cover the behavior with these two canister_status response variants.

I don't see any new test: did you push the test?

jedna commented:
@mraszyk https://github.com/dfinity/orbit/pull/538/files#diff-e825cffb84cec6b7f569564617db4e3bbcd0265eff65e9137e4308a3274e30d4R30-R49

mraszyk commented:
> @mraszyk https://github.com/dfinity/orbit/pull/538/files#diff-e825cffb84cec6b7f569564617db4e3bbcd0265eff65e9137e4308a3274e30d4R30-R49
> 
> Edit: But maybe you also meant a test for the external canister page backend itself, to see that the right panel is working properly. The lines above only cover the UI for modifications of LogVisibility, even though it's using the same source of data.

What I meant is an integration test for the station's `canister_status` endpoint. This was the first thing to fail in the reported issue.
